packages/arb-avm-cpp/avm_values/src/vmValueParser.cpp

```diff
@@ -25,18 +25,23 @@ const char* TUP_VAL_LABEL = "Tuple";
 const char* CP_VAL_LABEL = "CodePoint";
 const char* CP_INTERNAL_LABEL = "Internal";
 const char* OPCODE_LABEL = "opcode";
+const char* OPCODE_SUB_LABEL = "AVMOpcode";
 const char* IMMEDIATE_LABEL = "immediate";
 const char* CODE_LABEL = "code";
 const char* STATIC_LABEL = "static_val";
```

Comment:
curious why char* instead of string?

Reply:
No particularly good reason. Now that you mention it, I think `const std::string` would be better. I changed this here and a couple other places